docs/reference/semaphore-changelog.md

```diff
@@ -4,6 +4,18 @@ Thank you for using Semaphore!
 We continuously deploy changes that improve the product for you.
 This page is updated on a weekly basis.
 
+### Week of January 13, 2020
+- Updates to the macOS image:
+    - `macos-mojave-xcode11` Xcode 11.3.1 installed, to switch version use
+    `xcversion select 11.3.1`, image spec [macOS Mojave Xcode11](https://docs.semaphoreci.com/article/162-macos-mojave-xcode-11-image)
```

Comment:
I think it would be better to use new links, not help scout's - https://docs.semaphoreci.com/ci-cd-environment/macos-mojave-xcode-11-image/